Podfile

```diff
@@ -6,7 +6,7 @@ secrets = ["RIAM_CONFIG_URL", "RIAM_APP_SUBSCRIPTION_KEY"]
 target 'RInAppMessaging_Example' do
   pod 'RInAppMessaging', :path => '.'
   pod 'SwiftLint', '~> 0.42'
-  pod 'RSDKUtils', '~> 3.0.0', :testspecs => ['Nimble', 'TestHelpers']
+  pod 'RSDKUtils', :git => 'https://github.com/rakutentech/ios-sdkutils', :testspecs => ['Nimble', 'TestHelpers']
```

Comment:
The improvement relies on the unreleased changes in RSDKUtils

Reply:
To be honest I'm not sure what's the best approach. Update Podspec and Package.swift to use master branch version of RSDKUtils, wait for RSDKUtils release, or temporarily bring new SDKUtils changes into IAM SDK?

Reply:
@imaginaris I think it is better to keep RSDKUtils changes in IAM SDK until the release of RSDKUtils.
When will RSDKUtils be released?

Reply:
@malayli There's no plan for release yet

Reply:
@malayli Done. (SDKCF-5284)